### ISSUE TITLE
TNT 1.6.0  Release Crowdfunding

### ISSUE THREAD

**Issue by PyroSamurai** [TricksterOnline/TNT#7]
I'm giving this a second try since I don't like leaving things unfinished.
**Goal: $500** (arbitrary number, doesn't actually pay for all the time I will need for this)
**Currently raised: $503**
**Place to "donate":** https://paypal.me/PyroSamurai
Make sure to add a note for the payment's purpose like "TNT 1.6.0"

Update contents:
Well to start off with, it will have much simpler compile. I've made a batch script which, assuming you've already installed the JDK (which I've also made easier to install for Windows users), you just have to double-click and presto: TNT.jar is made.

Also I remove the hard-coded config file name. You can name them whatever you want now. I've moved that info inside the config file. 

Now that I think about it the list is quite extensive, but I'll try to list the ones I remember.
Keep in mind that the release isn't done yet:
- I've made the labels for the config files customizable.
- I've added auto-fix/calculation of size values
- I've made it so it autofixes photoshop bmp, & intake most BMP images as long as they are RGB555 or RGB24
- I've readded main support for subsets due to realizing that the majority aren't full of duplicates (whoops)
- I'm adding the ability to zlib-compress (like a .bac file) created NORI files
- I've added true support the extraData section as you can see from the updated specification file. I need more info to completely flesh that out though, but what I have done will allow completely functional NORI files to be made.
- I've made many code improvements as well, that as users you probably don't care about.

Still no 8-bit support, yet. I can only view those with GIMP, which is why that has been delayed. Imagine having to check every file in Photoshop, same thing. Maybe next release.

Files like those in the mapbgeffects aren't supported either.
I simply don't have enough info on their data structure.

Basically it is a quality of life update, + a major support boost for animated sprites.
This should be enough info for you to decide whether to chip in or not.
Any extra will go toward bug fixes and maintenance.

**Comment by PyroSamurai** [comment]
Thanks for the donation Daniel. 03/16/2023 + 04/07/2023

**Comment by PyroSamurai** [comment]
Thanks for the donation 瑋誠 林. 07/28/2023

**Comment by PyroSamurai** [comment]
Thanks for the amazing donation @grawlinson 4/14/2024

**Comment by PyroSamurai** [comment]
The 1.6.0 release has been fully funded ($500). Development will begin immediately, expected release date is 5/1/2024.
Any further updates or changes to the scheduled release will be mention in this thread.
Thank you donaters for your support.
